repo/packages/N/namerd/0/marathon.json.mustache

```diff
@@ -0,0 +1,63 @@
+{
+  "id": "{{namerd.name}}",
+  "instances": {{namerd.instances}},
+  "cpus": {{namerd.cpus}},
+  "mem": {{namerd.mem}},
+  "acceptedResourceRoles": [ "{{namerd.resource-role}}" ],
```

Comment:
Possible. namerd provides a REST API to control naming configuration across your cluster. It persists this data in ZK. We've found it convenient to have a namerd on `slave_public` to demonstrate API usage.